contrib/traceroute/traceroute.8

```diff
@@ -66,6 +66,22 @@ default.
 Firewall evasion mode.
 Use fixed destination ports for UDP, UDP-Lite, TCP and SCTP probes.
 The destination port does NOT increment with each packet sent.
+.It Fl E
+Detect ECN bleaching.
+Set the
+.Em IPTOS_ECN_ECT1
+Explicit Congestion Notification (ECN) bits
+.Pq Dv 01 ,
+and report if the hop has bleached
+.Pq Dv 00
+or mangled
+.Pq Dv 10
+them, or if it is experiencing congestion
+.Pq Dv 11 .
+Otherwise, report that it passed the bits appropriately.
+If
+.Fl t
+is also specified, the corresponding ECN bits will be replaced.
```

Comment:
Why do you say "replaced"? From my reading of the code, if `-t` is also specified we OR the ECN flag into the corresponding IP header field.

Reply:
In the third iteration, I zero the ECN bits before `OR`ing:
- IPv4:
   https://github.com/freebsd/freebsd-src/blob/6c00b180fedf9c4896dbfe523a5bb5235ade299f/contrib/traceroute/traceroute.c#L793-L794
- IPv6:
   https://github.com/freebsd/freebsd-src/blob/6c00b180fedf9c4896dbfe523a5bb5235ade299f/usr.sbin/traceroute6/traceroute6.c#L593-L598

The final crafted DSCP+ECN (originally ToS) (IPv4) or Traffic Class (IPv6) field when `-E` is passed should always be:
```
+-+-+-+-+-+-+-+-+
| Traffic Class |
+-+-+-+-+-+-+-+-+
|x x x x x x|0 1|
+-+-+-+-+-+-+-+-+
```

Reply:
I get it now, thank you.